ci/workflows/pull-request.yml

```diff
@@ -253,7 +253,7 @@ steps:
     retry: *auto-retry
 
   - label: "end-to-end iceberg sink v2 test"
-    if: build.pull_request.labels includes "ci/run-e2e-iceberg-sink-tests" || build.env("CI_STEPS") =~ /(^|,)e2e-iceberg-sink-tests?(,|$$)/
+    if: build.pull_request.labels includes "ci/run-e2e-iceberg-tests" || build.env("CI_STEPS") =~ /(^|,)e2e-iceberg-sink-tests?(,|$$)/
```

Comment:
Note that previously we use `ci/run-e2e-iceberg-sink-v2-tests` only for main-cron, but not here... It's confusing.

I think only `ci/run-e2e-iceberg-tests` is enough